tests/integration_tests/debezium/sql/debezium/real_test.sql

```diff
@@ -1,3 +1,5 @@
+-- set sql_mode REAL_AS_FLOAT is necessary
```

Comment:
Why set REAL_AS_FLOAT is neccesary?

Reply:
Because Debezium treats REAL as a synonym for FLOAT. https://github.com/pingcap/tiflow/blob/45b439b210688a28afc46bcb47fb24c81b02d6c7/pkg/sink/codec/debezium/codec.go#L385-L386